src/main/java/teammates/storage/api/FeedbackSessionsDb.java

```diff
@@ -542,41 +541,46 @@ public void deleteFeedbackSessionsForCourses(List<String> courseIds) {
     }
     
     @SuppressWarnings("unchecked")
-    private List<FeedbackSession> getFeedbackSessionEntitiesWithUnsentOpenEmail() {
+    private List<FeedbackSession> getFeedbackSessionEntitiesNeedingOpenEmail() {
         Query q = getPm().newQuery(FeedbackSession.class);
-        q.declareParameters("boolean sentParam, Enum notTypeParam");
-        q.setFilter("sentOpenEmail == sentParam && feedbackSessionType != notTypeParam");
+        q.declareParameters("java.util.Date startTimeParam, boolean sentParam");
+        q.setFilter("startTime > startTimeParam && sentOpenEmail == sentParam");
```

Comment:
Either method name is not 100% accurate:
- `getFeedbackSessionsWithUnsentOpenEmail()`: nowhere does it imply that date is part of the filter
- `getFeedbackSessionsNeedingOpenEmail()`: may be misleading because not-yet-opened sessions are included

Same problems apply for closing/closed email.